### PR TITLE
Feature branch - finished mint-sign up replica

### PR DESCRIPTION
**Pull request by Yors-git**
It contains: 

- Formatted buttons at the top and in the form
- A form with inputs with appropriate types and labels
- Footer
- Proper layout of the sign up page

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+      <fieldset class="main-form-fields flex">
+        <label for="email">
+          Email address
+          <input type="email" name="email">
+        </label>
+        <label for="phone">
+          Phone (recommended)
+          <input type="text" name="phone">
+          <span id="disclaimer">Standard call, messaging or data rates may apply.</span>
+        </label>
```
Comment (canriquez) [inline index.html]: - Please use the right type for the specific input information. 
image
hint from [w3schools.com](https://www.w3schools.com/tags/att_input_type_tel.asp)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+      <section id="legend-header" class="flex">
+        <h3>Create an Intuit Account</h3>
+        <h5>One account for everything Intuit, including Mint.</h5>
+        <a href="#" class="link">Learn more</a>
+      </section>
+      <fieldset class="main-form-fields flex">
+        <label for="email">
+          Email address
+          <input type="email" name="email">
+        </label>
```
Comment (canriquez) [inline index.html]: For all the label-input tag patterns, make sure that the for="" property in the label tag, match with an id="" property used in the input.
Hint: 
image

**Review by canriquez — CHANGES_REQUESTED** [review]
Hello 👋  Jorge Torres Bribiesca & Alaukik Khare. Project needs changes  ♻️
--

Good work so far in this project 👍 . 
You've produced a good read me file. Great use of  Github Flow, with good commit information and detailed pull request. Also, the page looks really good and very similar to the original. 
There are still some points to address so you can continue forward to the next project. Please work on the items below and submit a code review request when you finish ticking on the boxes in the list below.


- [x]  Add .gitignore file and include .DS_Store files. This file is not related to the project. 
image

- [x] FIx label tags - input-tags patterns. Hint from [Learn to Code HTML-CSS](https://learn.shayhowe.com/html-css/building-forms/)
image
 

- [x] Use the specific input tag type for the requested data in the form. Hint: check the phone input tag and use the correct type for it.
image


- [x] Fix the liner issues in this submission. The project has linter problems. Check commented code below for more hints.

OPTIONAL: You don't need to make this changes to approve this project. We encourage you to start using HTML semantic to organize HTML sections. Major navigation links could be organized using NAV tag and also main content using MAIN tags.

Again, congratulations on the progress so far.  Work on those pending things and submit a new code review request. 

All the best.
Carlos

--
Carlos Anriquez | [github](https://github.com/canriquez) | [Twitter](https://twitter.com/cranriquez) | [LinkedIn](https://www.linkedin.com/in/carlosanriquez/) | [Portfolio](https://www.carlosanriquez.com/)

File: styles/style.css [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+  padding-bottom: 0.2rem;
+}
+
+#button-text {
+  vertical-align: middle;
+  font-size: 15.75px;
+  padding-left: 0.5rem;
+}
+
+#disclaimer {
```
Comment (andresporras3423) [inline styles/style.css]: Don't overuse ID selectors

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
### Status: Changes Required :recycle:
**Hello @Yors-git @newhorizon-tech** 
Good job with your new submission for this project :muscle:  :muscle:
Your page passes most of the requirements :clap: :clap: :clap:  
There are just a few details to improve before you can go ahead:
- [x] Don't use important
- [x] The images below the Intuit logo are not in the correct order:

| Yours | Original |
|:---:|:---:|
|image | image |

- [x] The input password in the form should have a lock icon:

| Yours | Original |
|:---:|:---:|
|image | image |

Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Hey team,  your project it's looking really good, however, in order to accomplish as the actual page, you required to do these changes, listed below:
 - [ ] Your form is broken, kindly fix that.
<table>
 <tr>
 <th>Yours</th>
 <th>Original</th>
 </tr>
 <tr>
 <td>
</td>
 <td></td>
 </tr>
</table>
 
**Please make the required changes and resubmit for a new code review once you are ready**

As always, I wish you all the very best and see you both at the top🚀

  

<hr>

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

<sub>*We're here to help you, to ensure that you're using the best practices and getting you ready for the job market. In real-world, you'll have a lot of changes requests for your real clients, it's good to get used to it.</sub>

**Review by canriquez — APPROVED** [review]
Hello 👋  Jorge Torres Bribiesca & Alaukik Khare. Project Approved!  ✅
--
Congratulations on a really well-made project. The page looks great 🎉
You have implemented all suggested changes and TSE indications. The code now is clean and complying in full with all the requirements. Also, linters look all green this time 👍.

You are now ready to go for bigger challenges!

Keep up the good work.
Best regards
Carlos

Carlos Anriquez | [github](https://github.com/canriquez) | [Twitter](https://twitter.com/cranriquez) | [LinkedIn](https://www.linkedin.com/in/carlosanriquez/) | [Portfolio](https://www.carlosanriquez.com/)

**Pull request merged**